explorer/client/src/pages/transaction-result/TransactionResult.tsx

```diff
@@ -39,33 +104,46 @@ function TransactionResult() {
         );
     }
 
-    const data = findDataFromID(txID, state);
-
-    if (instanceOfDataType(data)) {
-        let action: string;
-        let objectIDs: string[];
-
-        if (data.created !== undefined) {
-            action = 'Create';
-            objectIDs = data.created;
-        } else if (data.deleted !== undefined) {
-            action = 'Delete';
-            objectIDs = data.deleted;
-        } else if (data.mutated !== undefined) {
-            action = 'Mutate';
-            objectIDs = data.mutated;
+    if (isCertifiedTransaction(showTxState)) {
```

Comment:
This is not a good pattern. The data validation already happens at the SDK level, so we should not need to check again here. If `data.loadState == 'loaded'`, you can then safely assume that `txState` is `CertifiedTransaction`